### PR TITLE
3 util functions for main so the ui looks nicer and 1 bug fix

### PR DESCRIPTION
**Pull request by notcooler**
# Description

i added 3 util functions in the main file. getPlatform(), setConsoleSize(), and clear()

getPlatform returns a string according to the platform.
setConsoleSize sets console size by the following x and y values. used in the begin so the banner looks nicer :)
clear() called before banner prints so it also looks more nicer. 👍 

# Issue Fixes
There was one issue if the user was using python 3.9 or above and that was that match stament wasnt used anymore in subreddit.py line 44, so changed it to a simple understandable if line. Hope it helps!

# Checklist:

- [ y] I am pushing changes to the **develop** branch
- [y ] I am using the recommended development environment
- [ y] I have performed a self-review of my own code
- [ y] I have commented my code, particularly in hard-to-understand areas
- [ y] I have formatted and linted my code using python-black and pylint
- [ y] I have cleaned up unnecessary files
- [ y] My changes generate no new warnings
- [y ] My changes follow the existing code-style
- [y ] My changes are relevant to the project

# Any other information (e.g how to test the changes)
Hope i sent enough info. ;)
run and see the changes

### CODE REVIEW

**Comment by StopmotionSimonYT** [comment]
Looks good! But can you remove the fix of the match statement? It is already fixed in the develop branch and now it is causing merge conflicts.

**Review by AcidClown — APPROVED** [review]

**Comment by notcooler** [comment]
all done is it good now?

**Comment by StopmotionSimonYT** [comment]
It has still merge conflicts, can you figure out what it is? I can also copy your pull request and fix it (ofcourse give you the credit).

**Comment by notcooler** [comment]
fixed it now

**Comment by notcooler** [comment]
it says i merged develop to master, but i didnt do idk what that is

**Comment by StopmotionSimonYT** [comment]
> it says i merged develop to master, but i didnt do idk what that is

You synchronized the branches. Now it works! @JasonLovesDoggo or @callumio can it be merged?

**Comment by JasonLovesDoggo** [comment]
> > it says i merged develop to master, but i didnt do idk what that is
> 
> You synchronized the branches. Now it works! @JasonLovesDoggo or @callumio can it be merged?

I will leave it to Callum

**Comment by notcooler** [comment]
Is callum busy or... why isnt he responding

**Review by StopmotionSimonYT — APPROVED** [review]

**Comment by StopmotionSimonYT** [comment]
> Is Callum busy or... why isn't he responding?

Due to personal circumstances, he is not responding

**Comment by JasonLovesDoggo** [comment]
> > Is Callum busy or... why isn't he responding?
> 
> Due to personal circumstances, he is not responding

@StopmotionSimonYT did you actually test it to see it looks fine or just review the code..

**Comment by StopmotionSimonYT** [comment]
> > > Is Callum busy or... why isn't he responding?
> > 
> > Due to personal circumstances, he is not responding
> 
> @StopmotionSimonYT did you actually test it to see it looks fine or just review the code.. 

I just reviewed the code but I could test it if you want.

**Comment by StopmotionSimonYT** [comment]
> > > @StopmotionSimonYT did you actually test it to see it looks fine or just review the code.. 
> > 
> > I just reviewed the code but I could test it if you want.
> 
> Its simple tho i think you get it by just looking at it- althougj test it if you want

Yes the code looks good to me! But can you move the defs to the utils folder and import them in main.py? If that's done I would test it and it could be merged.

**Comment by notcooler** [comment]
it should work now. although @StopmotionSimonYT you gotta test it on windows tho since idk if putted the column and line values at the good place for windows.

**Review by StopmotionSimonYT — CHANGES_REQUESTED** [review]

**Comment by StopmotionSimonYT** [comment]
The resize resolution is not good on Windows
image

File: main.py [hunk 1]
```diff
@@ -35,7 +46,7 @@
 )
 # Modified by JasonLovesDoggo
 print_markdown(
-    "### Thanks for using this tool! [Feel free to contribute to this project on GitHub!](https://lewismenelaws.com) If you have any questions, feel free to reach out to me on Twitter or submit a GitHub issue. You can find solutions to many common problems in the [Documentation](): https://reddit-video-maker-bot.netlify.app/"
+    "### Thanks for using this tool! [Feel free to contribute to this project on GitHub!](https://lewismenelaws.com) If you have any questions, feel free to reach out to me on Twitter or submit a GitHub issue. You can find solutions to many common problems in the [Documentation](https://luka-hietala.gitbook.io/documentation-for-the-reddit-bot/)"
```
[the remark this answers is not in the document]
Comment (notcooler) [reply main.py:50]: I didnt do that tho

**Comment by notcooler** [comment]
Anyways i decided to just cancel this rewuest since you guys are developing a gui system as well so i dont really want to work on this anymore. But thanks for all the help. :)

PS - Also sorry i was on vacation i couldnt use github for a while

**Pull request closed without merging**